Softeq.XToolkit.DefaultAuthorization/SessionApiService.cs

```diff
@@ -103,7 +108,7 @@ public async Task<ExecutionStatus> RefreshTokenAsync()
                         tokens.AccessTokenExpirationTimespanInSeconds);
                     result = ExecutionStatus.Completed;
                 }
-                else if (noInternetCodes.Contains(response.StatusCode))
+                else if (response.IsNoInternet || noInternetCodes.Contains(response.StatusCode))
```

Comment:
Should we even check noInternetCodes here? Looks like response itself, or rather HttpServiceGate, should determine if there's no internet internally.. Or is this logic not implemented properly?